### DISCUSSION TITLE
Create function to define media fluxes from concentrations

### DISCUSSION THREAD

**Discussion by NantiaL** [draeger-lab/refinegems#110] [Ideas]
It would be very useful to create a function to calculate fluxes needed for _in silico_ media definitions from concentrations given the respective molecular weights. 
The input could be the media definition and the function should systematically calculate the fluxes of needed exchanges. This calculation would take into account the proportion of each molecule, their respective charges, and the molecular weights of the solutions present in the medium.

A draft has already been made and remains subject to further refinement to achieve full automation.

**Comment by NuruddinKhoiry** [comment]
Hi there,

Has this function been implemented?
Also, I am really curious about the full explanation of this from the hint: "A full explanation for converting concentration to flux will be coming soon." When will it be ready?

Thank you

Khoiri
